README.md

```diff
@@ -71,4 +64,4 @@ The following diagram shows a deployment view of the project:
 
 # Footnotes
 
-<a id="thread-note">1</a>: This is slightly misleading, as it suggests that there is only one SSE program running. Actually, each client gets its own thread, but they are all running the same program. The client can pass data (e.g. an id) to the SSE script through the query string so that it can behave differently for each client. [↩](#thread-note-source)
+<a id="thread-note">1</a>: This is slightly misleading, as it suggests that there is only one SSE program running. Actually, each client gets its own thread, but they are all running the same program. The client can pass data (e.g. an id) to the SSE script through the query string so that it can behave differently for each client. [back](#thread-note-source)
```

Comment:
I didn't know you could use HTML tags in markdown! Pretty neat

Reply:
You can, but I’ll actually replace this with a pandoc footnote if we go this way. Using the html tag was just a workaround for github :)